02-peer-protocol.md

```diff
@@ -787,9 +787,22 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
 #### Requirements
 
 A sending node:
-  - MUST NOT offer `amount_msat` it cannot pay for in the
-remote commitment transaction at the current `feerate_per_kw` (see "Updating
-Fees") while maintaining its channel reserve.
+  - if it is _responsible_ for paying the Bitcoin fee:
+    - MUST NOT offer `amount_msat` if, after adding that HTLC to its commitment
+    transaction, it cannot pay the fee for either the local or remote commitment
+    transaction at the current `feerate_per_kw` while maintaining its channel
+    reserve (see [Updating Fees](#updating-fees-update_fee)).
+    - SHOULD NOT offer `amount_msat` if, after adding that HTLC to its commitment
+    transaction, its remaining balance doesn't allow it to pay the commitment
+    transaction fee when receiving a future additional non-dust HTLC while maintaining
+    its channel reserve ("fee spike buffer"). A buffer that expects the current
```

Comment:
This makes it sound like "channel reserve" == "fee spike buffer", while afaik "channel reserve" is only the reserve negotiated in open channel?

Reply:
I didn't understand what you mean at first, but I get it now. It sounds like what's inside the parenthesis complements what comes right before (channel reserve). I'll try to re-word that to avoid linking the two.

Reply:
I re-phrased in https://github.com/lightningnetwork/lightning-rfc/pull/740/commits/1ff808335dd3dc2f30e713d6ea98ed7297c354f3, let me know if that's better. I'm still not very happy with the last sentence...

Reply:
Yes, that resolves the confusion